pkg/console/vip.go

```diff
@@ -21,21 +25,23 @@ func createMacvlan(name string) (netlink.Link, error) {
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to fetch %s", name)
 	}
+	rand.Seed(time.Now().UnixNano())
+	macvlanName := tempMacvlanPrefix+strconv.Itoa(rand.Intn(100))
```

Comment:
What happens if we have a conflict name?  A random integer in 100 doesn't look safe

Reply:
I think 100 is far enough. After requesting the VIP, the temporary macvlan device will be deleted immediately. Unless users add network interfaces on purpose, there would not have a conflict network interface. With a very small probability, there will be an error message on the console if a conflict occurs.